readers.go

```diff
@@ -6,9 +6,9 @@ import (
 	"io"
 )
 
-// readIntoTree will read segments of size 'segmentSize' and push them into the
-// tree until EOF is reached.
-func readIntoTree(t *Tree, r io.Reader, segmentSize int) error {
+// ReadIntoTree will read segments of size 'segmentSize' and push them into the
+// tree until EOF is reached. Does not pad data.
+func ReadIntoTree(t *Tree, r io.Reader, segmentSize int) error {
```

Comment:
remember to align comments to 80 chars, in vim this can be achieved easily by highlighting the line using `V` and then pressing `gq` to format it.